client/src/ui/elements/ExpandablePopup.tsx

```diff
@@ -0,0 +1,59 @@
+import { AnimatePresence, motion } from "framer-motion";
+import React from "react";
+import { SecondaryPopup } from "../elements/SecondaryPopup";
+
+interface ExpandablePopupProps {
```

Comment:
think it would be better to put ExpandablePopup in the same place as the OSWindow, and have 2 different components, ExandableOSWindow and OSWindow, seems weird to have one component OSWindow with condition returning either one component or the other